en/docs/get-started/tutorials/create-your-first-webhook.md

```diff
@@ -4,9 +4,9 @@ Choreo’s low-code editor allows developers to easily design (and then implemen
 
 In this tutorial, you will address this requirement by doing the following:
 
-- Design a webhook that addresses the described requirement and test it in the Web Editor (i.e., the editing tool of Choreo), and commit the changes so that they are also available in the Choreo Console.
+- Create a webhook component by connecting to your GitHub repo that includes the implementation of the webhook which addresses the described requirement 
 - Deploy the webhook you created to the development environment.
-- Integrate GitHub with Choreo by connecting the webhook that you created to the required GitHub repository.
+- Modify the webhook implementation to connect to GitHub to act on GitHub related events.
```

Comment:
"to" is over-used. How about the following suggestion?

"Connect the webhook implementation to GitHub, enabling it to act in response to selected GitHub-related events."

Reply:
done